index.html

```diff
@@ -65,7 +65,131 @@ <h2 class="main-title main-section-subtitle">I'm a software<br> developer</h2>
             </section>
         </section>
     </main>
-
+    
     <!--end of main section-->
+
+    <!--Work section-->
+    <section class="work-section-container">
+        <h1 class="work-section-title">My Recent<br>Work</h1>
+        <hr class="hr-line">
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <div class="card-lable-container">
+                    <div class="card-lable">Ruby on rails</div>
+                    <div class="card-lable">css</div>
+                    <div class="card-lable">JavaScipt</div>
+                    <div class="card-lable">html</div>
+                </div>
```

Comment:
- [ ] Please use HTML lists for this part. You did a good job implementing the design, but one of the purposes for this template is for you to use `ul`'s and `li`'s and to be comfortable with them, so please change these `div`'s into the necessary `ul`'s and `li`'s tags.

_Note: Same thing applies for the rest of the cards_